app/helpers/xml.py

```diff
@@ -141,6 +141,14 @@ def build_mh_mets(g: rdflib.Graph, pid: str) -> str:
     for representation in representations:
         representation_index = representation.label.split("_")[1]
         representation_media = metsrw.FSEntry(type="Media")
+        representation_media.add_dmdsec(
+            build_minimal_sidecar(f"{pid}_{representation_index}"),
+            "OTHER",
+            **{
+                "othermdtype": "mhs:Sidecar",
+                "id": f"DMDID-MATERIALARTWORK-REPRESENTATION-{representation_index}",
+            },
+        )
```

Comment:
De metadata komt nu terecht op de metadata-only record object. Dat lijkt me de bedoeling?

Dit is wel minimale metadata. Wordt de metadata op de meemoo SIP representatie op een manier geplakt op de MH SIP?